imports/chrome/db.go

```diff
@@ -0,0 +1,41 @@
+package chrome
+
+import (
+	"github.com/bionic-dev/bionic/types"
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+)
+
+const dbRowSelectLimit = 100
+
+type DbURL struct {
```

Comment:
```suggestion
type DBURL struct {
```

Reply:
actually, do we need DB here? Maybe it's better to scope that part to `History`? Like that
```suggestion
type HistoryURL struct {
```

Reply:
ah ok it might be not only history in that Chrome's sqlite db. Then, how's about getting rid of `DB` and `History` and just don't use any prefix? 😄 
```suggestion
type URL struct {
```

Reply:
i just realized that i don't even use this struct, so it can be deleted altogether :)